pkgs/applications/misc/kemai/default.nix

```
@@ -0,0 +1,50 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, magic-enum
+, spdlog
+, qtbase
+, qtconnectivity
+, qttools
+, qtlanguageserver
+, wrapQtAppsHook
+, libXScrnSaver
+, nix-update-script
+}:
+
+stdenv.mkDerivation rec {
+  version = "0.9.2";
+  pname = "kemai";
+
+  src = fetchFromGitHub {
+    sha256 = "sha256-PDjNO2iMPK0J3TSHVZ/DW3W0GkdB8yNZYoTGEd2snac=";
+    rev = version;
+    repo = "kemai";
+    owner = "AlexandrePTJ";
+  };
+
+  buildInputs = [
+    qtbase
+    qtconnectivity
+    qttools
+    qtlanguageserver
+    libXScrnSaver
+    magic-enum
+    spdlog
+  ];
+  cmakeFlags = [ "-DUSE_CONAN=OFF" ];
+  patches = [ ./cmake.patch ];
+
+  nativeBuildInputs = [ cmake wrapQtAppsHook ];
+
+  passthru = { updateScript = nix-update-script { }; };
+
+  meta = with lib; {
+    description = "Kimai desktop client written in QT6";
+    homepage = "https://github.com/AlexandrePTJ/kemai";
+    license = licenses.mit;
+    maintainers = with maintainers; [ poelzi ];
+    platforms   = platforms.all;
```

```suggestion
    platforms   = platforms.unix;
```

Not all platforms have Qt yet.